<launch>
  <!--TOPICS' NAMES-->
  <param name="RANS_LINES_TOPIC" value="ransacLines"/>
  <param name="RANS_BISEC_TOPIC" value="ransacBisectrix"/>
  <param name="RANS_BISEC_PCL_TOPIC" value="bisectrix_pointcloud"/>
  <param name="VERO_LASER_TOPIC" value="/mkz/scan"/>
  <param name="VERO_ODOME_TOPIC" value="verocarfreedom/roda_odom"/>
  <!--param name="VERO_COMMA_TOPIC" value="verocarfreedom/cmd_vel"/-->
  <param name="VERO_COMMA_TOPIC" value="verocarfreedom/car_command"/>
  <param name="PION_COMMA_TOPIC" value="cmd_vel"/>

  <!--TF'S NAMES-->
  <!--VERO frame name-->
  <param name="LASE_FRAME_ID" value="/mkz/hokuyo_link"/>
  <!--Bagfile frame name -->
  <!--param name="LASE_FRAME_ID" value="hokuyo"/-->
  <param name="BASE_FRAME_ID" value="/mkz/base_link"/>
</launch>
